Apeswap-Bots/Gnana-fee-update/src/utils.ts

```diff
@@ -0,0 +1,46 @@
+import { Interface } from "@ethersproject/abi";
+import {
+  Finding,
+  FindingSeverity,
+  FindingType,
+  getEthersProvider,
+} from "forta-agent";
+
+interface MetaDataI {
+  previousFee: string;
+  currentFee: string;
+}
+const GNANA_TOKEN_ADDRESS: string =
+  "0xddb3bd8645775f59496c821e4f55a7ea6a6dc299";
+
+const EVENT_ABI: string[] = [
+  "event UpdateTaxFee(uint256 previousTaxFee, uint256 newTaxFee)",
+];
+
+const EVENTS_IFACE: Interface = new Interface(EVENT_ABI);
+
+
+const provider = getEthersProvider();
+
+const createFinding = (metaData: MetaDataI): Finding => {
+  return Finding.fromObject({
+    name: "Detect Fees Related To The Token",
+    description: "Token tax fee has been changed",
```

Comment:
```suggestion
    description: "GNANA Token tax fee has been changed",
```

(the same for the .spec findings)